spotbugs-ant/src/main/java/edu/umd/cs/findbugs/anttask/AbstractFindBugsTask.java

```diff
@@ -95,6 +95,8 @@ public String getValue() {
 
     public String execResultProperty = "edu.umd.cs.findbugs.anttask.AbstractFindBugsTask" + "." + RESULT_PROPERTY_SUFFIX;
 
+    private StringBuffer inputStringBuffer = null;
```

Comment:
If we need synchronization, then the methods that initialise this field should be synchronized.

If we don't need synchronization, it's cheaper to use a `StringBuilder`.

Reply:
I suspect that my original thinking was that, because these are not private methods, there was no way of knowing if some pre-existing caller might need these to be synchronized.   But yes, you are correct that I missed synchronizing the initialization.

Reconsidering it now, it makes more sense to push the synchronization to the caller if that is needed.   There's nothing formally declared that any of these methods are safe to call from multiple threads.  I'll change it to StringBuilder.